File: ClientApp/src/app/supplier/_interfaces/supplier-dto.ts
export interface SupplierDto {
    personId: number;
    personNumber: string;
    gender: boolean;
    firstName: string;
    lastName: string;
    dateOfBirth: Date;
    telephone: string;
    fax: string;
    countryId: number;

    countryName: string;
    continentName: string;
}

File: Data/Dto/Suppliers/SupplierDto.cs
using System;

namespace Shine.Data.Dto.Suppliers
{
    public class SupplierDto
    {
        public int PersonId { get; set; }
        public string PersonNumber { get; set; }
        public bool Gender { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Telephone { get; set; }
        public string Fax { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public int CountryId { get; set; }
    }
}
